src/lib/components/header/global-search-bar/GlobalSearchBar.svelte

```diff
@@ -364,6 +364,7 @@
     font-weight: 500;
     margin-right: 8px;
     text-align: left;
+    width: 40px;
```

Comment:
Can do it in percentage.

Reply:
this is fixed value for the icons. percentage is varying due to resize.